### PR TITLE
bt-434: Create chat screen

### LINKED ISSUE #434: Create chat screen
**User story:**
As a user, I want to have a chat screen, so that I can texting with potential employers.

**Acceptance criteria:**
- There should be a chat screen with messages in the middle with an input text area field at the bottom
- The text area field should be with the send message button
- The user’s profile photo and user name should be displayed on the top of the chat page with messages
- Allowed length for text area – up to 1000 characters
- When a user enters more than 1000 characters the validation message about the maximum size is displayed and the entered text should not be cut off

Image

### PR DESCRIPTION
**Pull request by anogire**
Based on [PR#554](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/pull/554)

### CODE REVIEW

**Comment by anogire** [comment]
Answers from Talents are provided for demonstration purposes only


https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/70234990/98436496-a654-44db-833c-7cf47d7d2640

File: mobile/src/bundles/chat/components/message-entry-field/message-entry-field.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                placeholder="Type a message"
+                multiline
+                maxLength={1000}
+                style={[
+                    globalStyles.flex1,
+                    globalStyles.ph15,
+                    styles.textEntryField,
+                ]}
+            />
+            <Pressable onPress={handleSendMessage}>
```
Comment (mskmee) [inline mobile/src/bundles/chat/components/message-entry-field/message-entry-field.tsx:75]: Maybe better to use TouchableOpacity here

**Comment by katerynakostikova** [comment]
Let's make a back button like an arrow 
image

**Comment by anogire** [comment]
Screenshot 2023-09-21 at 12 21 11

**Review by nikita-remeslov — APPROVED** [review]

**Review by katerynakostikova — APPROVED** [review]

**Pull request merged**